csm/cli/support_bundle/__init__.py

```diff
@@ -12,7 +12,3 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
```

Comment:
Minor: add the new line at the end of the file, please.

Reply:
updated.